Otus-DevOps-2018-02/ifqthenp_microservices
# HW 25. CI/CD with Kubernetes (kubernetes-4)

# Completed Homework №25


## The following tasks have been completed

- Installed and configured Helm
- Deployed Helm Charts with `ui`, `comment`, and `post` applications using template functions
- Deployed `reddit` Chart using Helm
- Deployed Gitlab Community Edition using Helm
- Created `ui`, `post`, `comment`, and `reddit-deploy` public repositories in Gitlab
- Created `.gitlab-ci.yml` config files for all repos

## How to get the project running

- Follow the instructions in Homework 25

## How to check that project actually works

- Ensure that the build passed TravisCI tests
  - [#andrei_bogomja](https://devops-team-otus.slack.com/messages/C9M6L1XB3) Slack channel
  - actual GitHub pull request page
- Make sure that on all stages of this homework application is working as described in the slides

## PR checklist
